example/service.js

```diff
@@ -1,8 +1,10 @@
-let indexd = require('indexd')
+//let indexd = require('indexd')
+let indexd = require('../index')
```

Comment:
I think for the example it makes sense to do `indexd`? No?

Reply:
oh you are right ...